### PR TITLE
Add trees list

### PR DESCRIPTION
**Pull request by jakub-drak-sbahi**
image

### CODE REVIEW

**Comment by BartoszChrostowski** [comment]
Is it hard to change those colors of icons? Are they svg?

File: web/src/app/tree/shared/trees-list-element.model.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export interface TreesListElement {
+  treeId: string,
+  treeName: string,
+  photoUri: string
```
Comment (BartoszChrostowski) [inline web/src/app/tree/shared/trees-list-element.model.ts:4]: Maybe there's a special type for URL?

File: web/src/app/tree/trees-list/trees-list.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+describe('TreesListComponent', () => {
+  let component: TreesListComponent;
+  let fixture: ComponentFixture<TreesListComponent>;
+  let trees: TreesListResponse = {
+    trees: [
+      {
+        treeId: "1",
+        treeName: "Tree 0",
+        photoUri: null
```
Comment (BartoszChrostowski) [inline web/src/app/tree/trees-list/trees-list.component.spec.ts:19]: Have u smoke tested it on some random?

File: web/src/app/tree/trees-list/trees-list.component.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            <h5 class="card-title text-center">{{ tree.treeName }}</h5>
+          </div>
+        </div>
+      </a>
+    </div>
+
+    <div class="col mb-4">
+      <a href="#" class="link-unstyled">
+        <div class="card" style="width: 18rem;">
+          <img class="card-img-top" style="width: 18rem; align-content: center;"
```
Comment (BartoszChrostowski) [inline web/src/app/tree/trees-list/trees-list.component.html:19]: Ok, I assume URLs are not supported?

**Review by BartoszChrostowski — APPROVED** [review]

**Comment by jakub-drak-sbahi** [comment]
> Is it hard to change those colors of icons? Are they svg?

Yes, they are svg. Is was not possible to choose color on site I downloaded it from.

File: web/src/app/tree/shared/trees-list-element.model.ts (hunk 1 above)
Comment (jakub-drak-sbahi) [reply web/src/app/tree/shared/trees-list-element.model.ts]: Changed

File: web/src/app/tree/trees-list/trees-list.component.spec.ts (hunk 2 above)
Comment (jakub-drak-sbahi) [reply web/src/app/tree/trees-list/trees-list.component.spec.ts:19]: Yes

**Pull request merged**